site/_js/slider.js

```diff
@@ -0,0 +1,183 @@
+import {gsap} from 'gsap';
+import {ScrollTrigger} from 'gsap/ScrollTrigger';
+import {ScrollToPlugin} from 'gsap/ScrollToPlugin';
+
+gsap.registerPlugin(ScrollTrigger, ScrollToPlugin);
+
+const cssClasses = {
+  NAVIGATION_WRAPPER: 'navigation-wrapper',
+  FIXED_NAVIGATION_WRAPPER: 'fixed-navigation-wrapper',
+  YEAR_NAVIGATION: 'year-navigation',
+  FIXED_YEAR_NAVIGATION: 'fixed-year-navigation',
+  MENU: 'menu',
+  FIXED_MENU: 'fixed-menu',
+  MENU_ITEM: 'menuitem',
+  MENU_ITEM_LINK: 'sectionLink',
+  CHROME_LOGO: 'chrome-logo',
+  YEAR_SECTION: 'inner-card',
+  HIDE_NAVIGATION: 'hide-navigation',
+};
+
+class Slider {
+  constructor() {
+    this.currentIndex = 0;
+    this.previousIndex = 0;
+
+    this.initialize();
+  }
+
+  initialize() {
+    window.addEventListener('load', () => {
+      const yearTimeline = gsap.timeline();
+      yearTimeline.add(this.startup());
+      yearTimeline.add(this.scrollSections());
+      yearTimeline.add(this.clickNavLink());
+    });
+
+    window.addEventListener('scroll', () => {
```

Comment:
Please make this is a [passive event listener](https://developers.google.com/web/updates/2016/06/passive-event-listeners), unless there is some reason that you can't. Additionally (and I'm not sure about this recommendation; it really depends on the use case), you might consider refactoring this to use an [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) if possible.

Reply:
> For the keyboard config, couple of things:
> 
> * When I press tab starting from the first page description, the cursor goes from "Scroll to begin" to 2008 and then continues down the years. I think we should make it such that it goes down to 2008 and then moves through the content cards within 2008, and then moves to the next year (and that year's content cards and so on)
> * Also, when you go down the years via keyboard, the page isn't scrolling up to show the last few years. I assume that problem won't be an issue if we use the above navigation path, but just please keep that in mind.

1. Reg your #1, we cannot set this behaviour using template as it is standard DOM flow.
2. We will look into it and check the possibilities to implement the same.